gitbook-docs/subscription_protocol.md

```diff
@@ -1,194 +1,198 @@
 
-* `period=[millisecs]` becomes the transmission rate, e.g. every `period/1000` seconds. Default=1000
+* `period=[millisecs]` becomes the transmission rate, e.g. every `period/1000` seconds. Default: 1000
 * `format=[delta|full]` specifies delta or full format. Default: delta
 * `policy=[instant|ideal|fixed]`. Default: ideal
- * `instant` means send all changes as fast as they are received, but no faster than `minPeriod`. With this policy the client has an immediate copy of the current state of the server.
+ * `instant` means send all changes as fast as they are received, but no faster than `minPeriod`. With this policy the
+     client has an immediate copy of the current state of the server.
  * `ideal` means use `instant` policy, but if no changes are received before `period`, then resend the last known
-   values.eg send changes asap, but send the value every `period` millisecs anyway, whether changed or not.
+   values.
  * `fixed` means simply send the last known values every `period`.
-* `minPeriod=[millisecs]` becomes the fastest message transmission rate allowed, e.g. every `minPeriod/1000` seconds. This is only relevant for policy='instant' to avoid swamping the client or network.
+* `minPeriod=[millisecs]` becomes the fastest message transmission rate allowed, e.g. every `minPeriod/1000` seconds.
+    This is only relevant for policy='instant' to avoid swamping the client or network.
+
+You can subscribe to multiple data keys multiple times, from multiple apps or devices. Each app or device simply
+subscribes to the data it requires, and the server and/or client implementation may combine subscriptions to avoid
+duplication as it prefers on a per connection basis. At the same time it is good practice to open the minimum
+connections necessary, for instance one WebSocket connection shared between an instrument panel with many gauges,
+rather then one WebSocket connection per gauge.
 
-You can subscribe to multiple data keys multiple times, from multiple apps or devices. Each app or device simply subscribes to the data it requires, and the server and/or client implementation may combine subscriptions to avoid duplication as it prefers on a per connection basis. At the same time it is good practice to open the minimum connections necessary, for instance one websocket connection shared bewteen an instrument panel with many gauges, rather then one websocket connection per gauge.
+## Multiple value handling in subscriptions
 
-#### Multiple value handling in subscriptions
+A subscription to a key is for all the updates to that key. If there are multiple sources generating data for that key
+the client will get all their updates.
 
-A subscription to a key is for all the updates to that key. If there are multiple sources generating data for that key the client will get all their updates.
+If a client wants only the values of a single source it should subscribe to a path that includes the full path under
```

Comment:
Duplication from data_model_multiple_values.md, maybe should be here and reference in multiple values?